Include/code.h

```diff
@@ -90,17 +90,17 @@ typedef struct {
 
 /* These are no longer used. */
 #if 0
-#define CO_GENERATOR_ALLOWED    0x1000
+#define CO_GENERATOR_ALLOWED    0x10000
 #endif
-#define CO_FUTURE_DIVISION      0x2000
-#define CO_FUTURE_ABSOLUTE_IMPORT 0x4000 /* do absolute imports by default */
-#define CO_FUTURE_WITH_STATEMENT  0x8000
-#define CO_FUTURE_PRINT_FUNCTION  0x10000
-#define CO_FUTURE_UNICODE_LITERALS 0x20000
-
-#define CO_FUTURE_BARRY_AS_BDFL  0x40000
-#define CO_FUTURE_GENERATOR_STOP  0x80000
-#define CO_FUTURE_ANNOTATIONS    0x100000
+#define CO_FUTURE_DIVISION      0x20000
```

Comment:
Changing constant values is not something done often in Python. Can you please add a comment explaning that these constant values changes in Python 3.9 to fix bpo-39562?

Same comment for `__future__.py`.

This change is backward incompatible and so must be documented at:
https://docs.python.org/dev/whatsnew/3.9.html#changes-in-the-python-api

Reply:
> This change is backward incompatible

You mean in case someone is *hardcoding* the values instead of using the exposed variables?

Reply:
Yes.

Reply:
Isn't that out of contract?

Reply:
I'm not saying that it's wrong to change constant values. I just asked **to document** the we changed constant values: that's what @isidentical did in his updated PR ;-)

Reply:
Agreed 👌